src/read_agent_logging.in

```diff
@@ -19,23 +19,53 @@ URL="${1:-http://localhost:24231/metrics}"
 # Default to 10s if unset, as a fail-safe mechanism.
 INTERVAL=${COLLECTD_INTERVAL:-10}
 
+# 1. If the Logging Agent uses REST.
 # The part of the input that sed looks at looks like:
-#   stackdriver_successful_requests_count{grpc="false"} 255
-#   stackdriver_failed_requests_count{grpc="false",code="400"} 14
+#   stackdriver_successful_requests_count{grpc="false",code="200"} 255
+#   stackdriver_failed_requests_count{grpc="false",code="401"} 25
 #   stackdriver_failed_requests_count{grpc="false",code="404"} 18
-#   stackdriver_ingested_entries_count 7050
-#   stackdriver_dropped_entries_count 25
-#   stackdriver_retried_entries_count{code="500"} 2
-#   stackdriver_retried_entries_count{code="501"} 3
-#
+#   stackdriver_failed_requests_count{grpc="false",code="500"} 23
+#   stackdriver_failed_requests_count{grpc="false",code="501"} 9
+#   stackdriver_ingested_entries_count{grpc="false",code="200"} 7050
+#   stackdriver_dropped_entries_count{grpc="false",code="401"} 750
+#   stackdriver_dropped_entries_count{grpc="false",code="404"} 480
+#   stackdriver_retried_entries_count{grpc="false",code="500"} 600
+#   stackdriver_retried_entries_count{grpc="false",code="501"} 290
```

Comment:
Same here. HTTP 501 is gRPC error 12 (unimplemented). gRPC error 1 would probably be HTTP 503 (service unavailable).

Reply:
Changed.